modules/tfm/tfm/boards/common/assert.c

```diff
@@ -7,14 +7,20 @@
 #include <pm_config.h>
 #include "autoconf.h"
 #include "region_defs.h"
-
+#include <zephyr/toolchain/common.h>
 
 #define IS_ALIGNED_POW2(value, align) (((value) & ((align) - 1)) == 0)
 
 /* Making sure the borders between secure and non-secure regions are aligned with the SPU regions */
-
 #if !(IS_ALIGNED_POW2(PM_TFM_NONSECURE_ADDRESS, CONFIG_NRF_SPU_FLASH_REGION_SIZE))
-#error "TF-M non-secure address start is not aligned on SPU region size"
+#pragma message "\n\n!!!Partition alignment error!!!"\
```

Comment:
I don't get it how will that help. The reference in the terminal doesn't make a lot of sense I think since it cannot be resolved. I can try to add a text mentioning  "Application Development -> Running applications with Trusted Firmware-M" so that someone can try to find the document I think.

Reply:
Ok. I think that is better than trying to explain it in this very condensed form here.
Maybe refer to "TF-M partition alignment requirements"?